### PR TITLE
Add Moonbeam mainnet network support

### PR DESCRIPTION
**Pull request by datradito**
Endpoints for verification:
Moonbeam Chain ID (1284) : https://rpc.api.moonbeam.network

Block Explorers:
https://docs.moonbeam.network/builders/tools/explorers/

CC: @rmeissner

### CODE REVIEW

**Comment by datradito** [comment]
Hi @rmeissner , how can I add you as a reviewer? Thanks.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/safe-deployments",
-  "version": "1.7.0",
+  "version": "1.4.0",
```
Comment (rmeissner) [inline package.json:3]: This should no be touched ;)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +0,0 @@
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence.
-* @gnosis/safe-services
```
Comment (rmeissner) [inline .github/CODEOWNERS:3]: This should not be touched

File: src/assets/v1.0.0/gnosis_safe.json [hunk 3]
```diff
@@ -7,8 +7,7 @@
     "1": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
     "4": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
     "5": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
-    "42": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
-    "100": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
```
Comment (rmeissner) [inline src/assets/v1.0.0/gnosis_safe.json:11]: Why is Gnosis chain removed?

**Review by rmeissner — CHANGES_REQUESTED** [review]
Please rebase to the latest version, you are making changes that are related to old commits.

File: src/assets/v1.0.0/gnosis_safe.json (hunk 3 above)
Comment (datradito) [reply src/assets/v1.0.0/gnosis_safe.json:11]: It's seems was due a merge conflict. It's solved.

File: .github/CODEOWNERS (hunk 2 above)
Comment (datradito) [reply .github/CODEOWNERS:3]: It's seems was due a merge conflict. It's solved.

**Comment by datradito** [comment]
Hi @rmeissner , do I need to do something else? thanks

**Comment by rmeissner** [comment]
This PR is still making more changes than expected. It should only add the addresses for the moonbeam chain id, no other files should be touched and no other networks removed. Until this is resolved this cannot be merged.

**Comment by datradito** [comment]
> 

Perfect, I'm going to close this PR and I'll create a new one cleaner.

**Comment by datradito** [comment]
I'll create a more cleaner PR

**Pull request closed without merging**